jest.config.mjs

```diff
@@ -32,6 +32,7 @@ const config = {
   moduleNameMapper: {
     '^@/(.*)$': '<rootDir>/src/$1',
     '^uuid$': 'uuid',
+    "^jose": 'jose',
```

Comment:
https://stackoverflow.com/questions/76608600/jest-tests-are-failing-because-of-an-unknown-unexpected-token-export

jwks-rsa brings triggers export problem when testing